runtime/convert.go

```diff
@@ -207,6 +207,7 @@ func BytesSlice(val, sep string) ([][]byte, error) {
 // Timestamp converts the given RFC3339 formatted string into a timestamp.Timestamp.
 func Timestamp(val string) (*timestamppb.Timestamp, error) {
 	var r timestamppb.Timestamp
+	val = fmt.Sprintf(`"%s"`, strings.Trim(val, `"`))
```

Comment:
We could use `strconv.Quote` here instead of `fmt.Sprintf`.

Reply:
I understand, thank you!